new_client/src/actions/authActions.js

```diff
@@ -0,0 +1,100 @@
+import {
+  USER_LOADED,
+  USER_LOADING,
+  AUTH_ERROR,
+  LOGIN_SUCCESS,
+  LOGIN_FAIL,
+  REGISTER_SUCCESS,
+  REGISTER_FAIL
+} from "./types";
+import axios from "axios";
+import { returnErrors } from "./errorActions";
+
+export const loadUser = () => (dispatch, getState) => {
+  dispatch({ type: USER_LOADING });
+
+  axios
+    .get("http://localhost:5000/api/user", tokenConfig(getState))
```

Comment:
Use the environment variable here. Also, The env variable defines the port as something else, and you have defined it as 5000. Hence, use the env. variable itself, for the base url of the server/API